cmd/build.go

```diff
@@ -224,6 +224,14 @@ func build(config *buildCommandConfig) error {
 		return err
 	}
 
+	deprecated, depErr := GetDeprecated(config.RootCommandConfig)
+	if depErr != nil {
+		return depErr
+	}
+	if deprecated != "" {
+		config.Warning.logf("Stack deprecated: %v", deprecated)
+	}
+
```

Comment:
You could put this inside the `GetDeprecated` function so you only need to handle the error